pkg/registry/apis/provisioning/register.go

```diff
@@ -463,6 +465,12 @@ func (b *APIBuilder) GetPostStartHooks() (map[string]genericapiserver.PostStartH
 				return err
 			}
 
+			// When starting with an empty instance -- swith to "mode 4+"
+			err = b.tryRunningOnlyUnifiedStorage()
+			if err != nil {
+				return err
```

Comment:
It seems a bit odd that it's something we do within provisioning. It seems to belong to the dualwrite or the storage domain.

Reply:
💯 this is just a question of timing.

I'll add a note for that -- goal now is getting our branch to have the behavior that will likely be out-of-the box in another month